distribution/resources/conf/default-micro-gw.conf.template

```diff
@@ -504,6 +504,15 @@
   timestampSkew = 5000
   # The value of the server header sent in the response by microgateway
   header="ballerina"
+  # The headers will be preserved in the request and the response without any intermediate change
```

Comment:
should change to something like `Preserve or override header values intercepted by the gateway`

Reply:
Done with https://github.com/wso2/product-microgateway/pull/1577/commits/959f9e322ddbce9b0318a3a402e3470eb0aaf237